SPECS/kernel/config

```diff
@@ -410,31 +407,34 @@ CONFIG_X86_MCELOG_LEGACY=y
 CONFIG_X86_MCE_INTEL=y
 CONFIG_X86_MCE_AMD=y
 CONFIG_X86_MCE_THRESHOLD=y
-# CONFIG_X86_MCE_INJECT is not set
+CONFIG_X86_MCE_INJECT=m
```

Comment:
used for testing and QA. Not an optimization, ignoring for now
https://cateee.net/lkddb/web-lkddb/X86_MCE_INJECT.html